.github/workflows/build-kops.yml

```diff
@@ -52,6 +52,9 @@ jobs:
           - "1.18"
           - "1.19"
           - "1.20"
+        platform:
+          - amd64
+          - arm64v8
```

Comment:
Only 1.19 and 1.20 kops versions have binaries for arm64 .. not sure if matrix strategy can skip those versions for arm64 ...

Reply:
I fixed this using matrix.exclude